src/scraping/collector.py

```
@@ -0,0 +1,123 @@
+        
+        # Basic scoring for MVP
+        if listing.get("location", {}).get("schools", []):
+            score += 10
+        if listing.get("location", {}).get("shopping", []):
+            score += 5
+        if listing.get("location", {}).get("transit", []):
+            score += 5
+            
+        return min(max(score, 0), 100)
+
+    def _get_market_trend(self, listing: Dict[str, Any]) -> float:
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)